app/javascript/components/Glossary.vue

```diff
@@ -177,7 +177,7 @@
           </td>
         </tr>
         <tr>
-          <td class="border border-teal-light pl-2">initiative/ organisation</td>
+          <td class="border border-teal-light pl-2">Initiative/ organisation</td>
           <td class="border border-teal-light pl-2">
```

Comment:
Addresses [ticket 79](https://unep-wcmc.codebasehq.com/projects/tradehub-navigator/tickets/79) 👍🏻

Reply:
I'm not sure I understand here... Ticket #79 was about having a capital "i" for the word "initiative", so this is done, isn't it ?

Reply:
Yep, this isn't a request for anything to change, just a note for my own/others' benefit